src/migmose/mig/nachrichtenstrukturzeile.py

```diff
@@ -0,0 +1,21 @@
+"""
+contains class for lines in mig tables
+"""
+
+from pydantic import BaseModel
+
+
+class NachrichtenstrukturZeile(BaseModel):
+    """
+    class for lines in mig tables
+    """
+
+    zaehler: str
```

Comment:
Kannst du an diese Attribute Beispiele in Docstrings ergänzen mit denen man eine Zeile im mig zuordnen könnte? Das würde es glaube ich viel klarer machen. Fast mehr als ein Text